components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -108,51 +75,51 @@ export default class SystemUsersDropdown extends React.PureComponent {
         };
     }
 
-    handleMakeActive = (e) => {
+    handleMakeActive = (e: {preventDefault: () => void}) => {
         e.preventDefault();
         this.props.actions.updateUserActive(this.props.user.id, true).
             then(this.onUpdateActiveResult);
     }
 
-    handleManageTeams = (e) => {
+    handleManageTeams = (e: {preventDefault: () => void}) => {
         e.preventDefault();
 
         this.props.doManageTeams(this.props.user);
     }
 
-    handleManageRoles = (e) => {
+    handleManageRoles = (e: {preventDefault: () => void}) => {
         e.preventDefault();
 
         this.props.doManageRoles(this.props.user);
     }
 
-    handleManageTokens = (e) => {
+    handleManageTokens = (e: {preventDefault: () => void}) => {
         e.preventDefault();
 
         this.props.doManageTokens(this.props.user);
     }
 
-    handleResetPassword = (e) => {
+    handleResetPassword = (e: {preventDefault: () => void}) => {
         e.preventDefault();
         this.props.doPasswordReset(this.props.user);
     }
 
-    handleResetEmail = (e) => {
+    handleResetEmail = (e: {preventDefault: () => void}) => {
         e.preventDefault();
         this.props.doEmailReset(this.props.user);
     }
 
-    handleResetMfa = (e) => {
+    handleResetMfa = (e: {preventDefault: () => void}) => {
         e.preventDefault();
         adminResetMfa(this.props.user.id, null, this.props.onError);
     }
 
-    handleShowDeactivateMemberModal = async (e) => {
+    handleShowDeactivateMemberModal = async (e: {preventDefault: () => void}) => {
         e.preventDefault();
         if (this.shouldDisableBotsWhenOwnerIsDeactivated()) {
             await this.props.actions.loadBots(
                 Constants.Integrations.START_PAGE_NUM,
-                Constants.Integrations.PAGE_SIZE,
+                parseInt(Constants.Integrations.PAGE_SIZE, 10),
```

Comment:
Can we do a cast here instead of `parseInt`? If we expect `PAGE_SIZE` is always going to be an integer then I would prefer not to add additional logic.